shared/AppInsightsCore/src/JavaScriptSDK/CoreUtils.ts

```diff
@@ -11,32 +11,29 @@ const strAddEventHelper = "addEventListener";
 const strDetachEvent = "detachEvent";
 const strRemoveEventListener = "removeEventListener";
 
-function _isTypeof(value: any, theType: string): boolean
-{
+const _isTypeof = (value: any, theType: string): boolean => {
     return typeof value === theType;
-}
+};
 
-function _isUndefined(value: any): boolean
-{
-    return value === undefined || _isTypeof(value, strUndefined);
-}
+const _isUndefined = (value: any): boolean => {
+    return _isTypeof(value, strUndefined) || value === undefined;
+};
 
-function _isNullOrUndefined(value: any): boolean
-{
+const _isNullOrUndefined = (value: any): boolean => {
     return (_isUndefined(value) || value === null);
 }
 
-function _hasOwnProperty(obj:any, prop:string): boolean {
+const _hasOwnProperty = (obj:any, prop:string): boolean => {
```

Comment:
I initially attempted to also add "strict" set to true and this change was an attempt to work around TS complaining about evaluated values being possible undefined. While I undid the strict setting I havn't removed this change.

Currently performing comparisons against previous and this change, if this increases the total size of the prod packages or the generated code usage looks suspect I'll revert these